listen.go

```diff
@@ -383,6 +383,12 @@ func (app *App) startupMessage(addr string, isTLS bool, pids string, cfg ListenC
 		fmt.Fprintf(out, "%sINFO%s Application name: \t\t%s%s%s\n", colors.Green, colors.Reset, colors.Blue, app.config.AppName, colors.Reset)
 	}
 
+	// TODO: Make this startup timeout log configurable
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
We may want to add this as a field in app.Config. I can see some services taking longer than 30s to start and then causing issues for users

Reply:
Coderrabit suggested a nice implementation for this, I'll add it asap